test/testScan.py

```diff
@@ -32,6 +31,7 @@ def testnumindices(self):
         else:
             self.assertEquals(len(self.data.optdone), 12 + self.extra)
 
+    @skipForParser('ORCA', "?")

```

Comment:
@langner Any reason this was added for ORCA? Were you working on these tests prior to my recent update to the ORCA data files?